### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by brenoxav**
## Math magicians: full website

- Develop a full website with several pages
- Create a website consisting of 3 pages: Home, Calculator, and Quote
- General layout matching the given wireframes
- Add custom styles to improve the look and feel
- Create route for each page with React Router
- Use navigation links for all 3 routes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +17,7 @@ Description of the app goes here.
 
 ## Live Demo
 
-[Live Demo Link](https://brenoxav.github.io/math-magicians)
+[Coming Soon](https://brenoxav.github.io/math-magicians)
```
Comment (Dipeshtwis) [inline README.md:20]: - [ ] I would recommend either deploy your project on Github pages and use the correct link or remove the live demo link to make the README professional. Currently, the link is showing me `404 page not found`.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @brenoxav, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Good use of Router, switch, and route.
:+1: There is a website consisting of 3 pages: Home, Calculator, and Quote.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (brenoxav) [reply README.md]: The link was removed.

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @brenoxav, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**